app/controllers/votes_controller.rb

```diff
@@ -0,0 +1,21 @@
+class VotesController < ApplicationController
+  def create 
+    @vote = Vote.new(
+      user_id: 1,
+      work_id: params[:work_id]
+    )
+
+    puts "WORK_ID IS: #{@vote.work_id}."
+    puts "USER_ID IS: #{@vote.user_id}."
+
+    if @vote.save
+      flash[:success] = "Successfully voted for work with title \"#{Work.find_by(id: params[:work_id]).title}\""
+      redirect_to works_path
+
+    else
+      flash[:error] = "A problem occured"
+
+      redirect_back(fallback_location: root_path)
```

Comment:
It would be good to give a little more information here about what went wrong, both for the user and for you as the engineer working on the product.